evaluator/variable.go

```diff
@@ -96,6 +147,13 @@ func parseVariable(val interface{}, varType string) hilast.Variable {
 		varType = HCL_STRING_VARTYPE
 	}
 
+	return varType
+}
+
+func parseVariable(val interface{}, varType string) hilast.Variable {
```

Comment:
__[gometalinter]__
* [staticcheck] argument varType is overwritten before first use (SA4009)

<!-- Comment by [SideCI](https://sideci.com) -->